test_data_reference.py

```
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+from src.infrastructure.data_reference import ExternalDataManager
+from datetime import datetime
+
+
+def test_external_data_manager():
```

Ruff (0.12.2)

Missing return type annotation for public function `test_external_data_manager`

Add return type annotation: `None`

(ANN201)